src/esp/assets/ResourceManager.h

```diff
@@ -314,18 +314,21 @@ class ResourceManager {
    * gfx::Drawable will be rendered.
    * @param lightSetup The @ref LightSetup key that will be used
    * for the added component.
+   * @param visNodeCache Cache for pointers to all nodes created as the result
```

Comment:
```suggestion
   * @param[out] visNodeCache Cache for pointers to all nodes created as the result
```

will mark the parameter as output-only in the generated docs. Same in the other cases below.